libbeat/tests/system/beat/beat.py

```diff
@@ -283,6 +283,27 @@ def log_contains(self, msg, logfile=None):
         except IOError:
             return False
 
+    def log_contains_count(self, msg, logfile=None):
```

Comment:
Since this duplicates most of `log_contains()`, I think `log_contains()` could just call this method and `return log_contains_count(...) > 0`.